specifications/api/swagger.yaml

```diff
@@ -277,6 +277,28 @@ x-walletName: &walletName
   minLength: 1
   example: Alan's Wallet
 
+x-poolMetadataSource: &poolMetadataSource
+  description: Pool metadata source. null -> no fetching, direct -> direct fetching, uri -> use SMASH server
```

Comment:
That could be a little clearer and with a bit more prose. This is our end-user docs, so if we need to put details, diagrams and explanations, that's the right place!